ml-agents/mlagents/trainers/ppo/trainer.py

```diff
@@ -217,7 +217,10 @@ def _update_policy(self):
         return True
 
     def create_tf_policy(
-        self, parsed_behavior_id: BehaviorIdentifiers, behavior_spec: BehaviorSpec
+        self,
+        parsed_behavior_id: BehaviorIdentifiers,
+        behavior_spec: BehaviorSpec,
+        create_graph: bool = False,
     ) -> TFPolicy:
         """
         Creates a PPO policy to trainers list of policies.
```

Comment:
Not sure what create_graph does. Can you add a comment?

Reply:
added. It sets the `create_tf_graph` flag in the `TFPolicy` constructor